network-api/networkapi/wagtailpages/pagemodels/customblocks/annotated_image_block.py

```diff
@@ -15,8 +17,27 @@ class AnnotatedImageBlock(ImageBlock):
         default=False,
         help_text='Would you like to use a wider image on desktop?'
     )
+    full_width_image = blocks.BooleanBlock(
+        required=False,
+        default=False,
+        help_text='Would you like to use a full width image?',
+    )
 
     class Meta:
         icon = 'image'
         template = 'wagtailpages/blocks/annotated_image_block.html'
         help_text = 'Design Guideline: Please crop images to a 16:6 aspect ratio when possible.'
+
```

Comment:
Function that prevents user from selecting wide and full_width, only will let them select one or the other